distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -305,7 +308,8 @@ type File
     / : (Text | File) -> File
     / self subpath = self.resolve subpath
 
-    ## Join two or more path segments together, normalizing the `..` and `.` subpaths.
+    ## GROUP Metadata
+       Join two or more path segments together, normalizing the `..` and `.` subpaths.
```

Comment:
Why is that `Metadata`?

The definition of the group says: `Metadata:  A property of the object (e.g. length, row_count).`. I see nothing like that here.

It seems more like a transformation, maybe it should be `Calculations` or `Selections`?